package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-react": "^6.10.2",
     "eslint": "^3.14.0",
     "extract-text-webpack-plugin": "~2.1.0",
+    "fs-extra": "^3.0.1",
```

Comment:
@mAiNiNfEcTiOn This would still break when specifying a different dist directory.

@asci I prefer `outputFile` from `fs-extra`, since it perfectly matches the function signature from `saveThemeScssFile`.